mkosi.md

```diff
@@ -1143,6 +1143,14 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   the same location, suffixed with `.pub` (as done by `ssh-keygen`). If this
   option is not present, `mkosi` generates a new key pair automatically.
 
+`SshAgent=`, `--ssh-agent=`
+
+: If specified as a path, use the given socket to connect to the ssh agent when
+  building an image and when connecting via `mkosi ssh` instead of hard-coding
+  a key. If specified as `true`, `$SSH_AUTH_SOCK` will be parsed instead (hint:
+  use `sudo` with `-E`). The keys listed by `ssh-add -L` will be installed as
+  authorized_keys in the built image.
```

Comment:
"authorized keys" (no underscore), or "`authorized_keys`"

Oh, so this does two things: first it uses public keys to initialize `authorized_keys`, and then it uses the agent when connecting over ssh. I think the description should make it clearer.

Reply:
ok, updated